### DISCUSSION TITLE
(missing) Data in csv export?

### DISCUSSION THREAD

**Discussion by bjberlin** [mytourbook/mytourbook#1190] [Q&A]
Hi community, hi Wolfgang,

I did export data from selected tours. I can find altimeter going up but I am looking for altimeter data down. 
Is there any way to modify the columns to be exported? I can't find it :-/

Using MT 22.8.0

THX
Björn

**Comment by bjberlin** [comment]
Hi friends,
Now I have a csv with Excel columns up to "CT" - quite a lot. There is Elevation up and down in that file now.

First I used the statistic -> csv screen where there is only the data for uphill. Now I found a right click in the Tour List -> "export to csv" which gives me a lot of data, and of course the data for going down as well.
I expected this data to be somewhere available....

Björn

![MyTourbook](https://github.com/mytourbook/mytourbook/assets/146870666/9e74c917-04a9-49c5-9cfb-fe64c25dcbd3)

**Comment by wolfgang-ch** [comment]
The downhill statistic graph will be available in the next version 23.10 https://github.com/mytourbook/mytourbook/issues/1034#issuecomment-1691244212 and the CSV export will also include these values
